src/rustup/config.rs

```diff
@@ -93,7 +107,8 @@ impl Cfg {
             gpg_key: gpg_key,
             notify_handler: notify_handler,
             env_override: env_override,
-            dist_root_url: dist_root_url,
+            dist_root_url: dist_root,
```

Comment:
This patch looks great now, but is it possible to remove this `dist_root_url` field completely? Looks like it's mainly used by `rustup_dist::dist`, so that module can just be changed to append "/dist".